api-docs/paths/organizations/sessions.json

```diff
@@ -0,0 +1,187 @@
+{
+  "get": {
+    "tags": ["Organizations"],
+    "description": "Returns a time series of release health session statistics for projects bound to an organization.",
+    "operationId": "EXPERIMENTAL: Retrieve Release Health Session Statistics",
+    "parameters": [
+      {
+        "name": "organization_slug",
+        "in": "path",
+        "description": "The slug of the organization.",
+        "required": true,
+        "schema": {
+          "type": "string"
+        }
+      },
+      {
+        "name": "project",
+        "in": "query",
+        "description": "The ID of the projects to filter by.\nUse `-1` to include all accessible projects.",
+        "required": true,
+        "schema": {
+          "type": "array",
+          "items": {
+            "type": "integer"
+          }
+        },
+        "style": "form",
+        "explode": true
+      },
+      {
+        "name": "environment",
+        "in": "query",
+        "description": "The name of environments to filter by.",
+        "required": false,
+        "schema": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "style": "form",
+        "explode": true
+      },
+      {
+        "name": "field",
+        "in": "query",
+        "description": "The list of fields to query.\nThe available fields are `sum(session)`, `count_unique(user)`, and the following functions applied to the `session.duration` metric: `avg`, `p50`, `p75`, `p90`, `p95`, `p99` and `max`.\nFor example, `p99(session.duration)`.",
+        "required": false,
+        "schema": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "style": "form",
+        "explode": true
+      },
+      {
+        "name": "groupBy",
+        "in": "query",
+        "description": "The list of properties to group by.\nThe available groupBy conditions are `project`, `release`, `environment` and `session.status`.",
+        "required": false,
+        "schema": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "style": "form",
+        "explode": true
+      },
+      {
+        "name": "query",
+        "in": "query",
+        "description": "A free-form query that is applied as a filter.\nAn example query could be `release:\"1.1.0\" or release:\"1.2.0\"`.",
+        "required": false,
+        "schema": {
+          "type": "string"
+        }
+      },
+      {
+        "name": "interval",
+        "in": "query",
+        "description": "This is the resolution of the time series, given in a `\"<number><unit>\"` format.\nFor example `1h` for a one hour resolution. Possible units are `m` for minutes, `h` for hours, `d` for days and `w` for weeks.\nThe default resolution is `1h` and the minimum resolution is currently restricted to `1h` as well.",
```

Comment:
Should we mention here that the interval longer than 1d might lead to unexpected results? I don't remember what was to problem exactly.